asv_bench/benchmarks/plotting.py

```diff
@@ -111,22 +116,49 @@ class BackendLoading:
     warmup_time = 0
 
     def setup(self):
-        dist = pkg_resources.get_distribution("pandas")
```

Comment:
this test was invalid because the underlying code uses `importlib.metadata` and not `pkg_resources` - so this `pkg_resources.get_entry_map` monkeypatch was unused.

It only passed because `_get_plot_backend` falls back to `importlib.import_module`